lib/core/datatable/components/datatable/datatable.component.ts

```diff
@@ -221,10 +221,6 @@ export class DataTableComponent implements AfterContentInit, OnChanges, DoCheck,
             this.resetSelection();
             this.emitRowSelectionEvent('row-unselect', null);
         }
-
-        if (this.isPropertyChanged(changes['sorting'])) {
```

Comment:
I think this is still necessary due the fact that only the sorting can change and we need to set the correct one in this case.